var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -155,13 +156,10 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage):
     # https://github.com/tensorflow/tensorflow/issues/33374
     depends_on("python@:3.7", type=("build", "run"), when="@:2.1")
 
-    # TODO: Older versions of TensorFlow don't list the viable version range,
-    # just the minimum version of bazel that will work. The latest version of
-    # bazel doesn't seem to work, so for now we force them to use min version.
-    # Need to investigate further.
-
+    # See .bazelversion
+    depends_on("bazel@5.1.1", type="build", when="@2.10:")
```

Comment:
Bazel versions are now pinned, only the pinned version is supported: https://github.com/tensorflow/tensorflow/commit/4665c2b80f8c47a44a51a76c534bc5bd1749834f